gradle/publish-bom.gradle

```diff
@@ -1,10 +1,41 @@
-publishing {
-    publications {
-        nebulaBom(MavenPublication) {
-            nebulaDependencyManagement.fromConfigurations { configurations.compile }
-            artifactId = project.name + '-bom'
-        }
+configurations {
+    bom {
+        extendsFrom compile
     }
 }
 
 bintray { pkg { publications += ['nebulaBom'] } }
+
+afterEvaluate {
```

Comment:
I'm not really comfortable with this much extra complexity in our gradle, for unknown upside.  If the boms are going to be super valuable can we make a concerted effort to make this simpler/encapsulated, otherwise, can we just close?